sdk/aqueduct/utils.py

```diff
@@ -125,7 +125,7 @@ def retention_policy_from_latest_runs(k_latest_runs: int) -> RetentionPolicy:
 REQUIREMENTS_FILE = "requirements.txt"
 BLACKLISTED_REQUIREMENTS = "aqueduct"
 
-UserFunction = Callable[..., pd.DataFrame]
+UserFunction = Callable[..., Any]
```

Comment:
Unrelated but just a thought: for the metric and check functions, if they return a non-traditional value like np.bool_, will we cast their return values into a bool for them? Or will we use the raw type - I think we cast atm.

Reply:
casting seems fine to me.